src/lib389/lib389/cli_conf/backend.py

```diff
@@ -907,77 +907,77 @@ def create_parser(subparsers):
     # List VLV Searches
     list_vlv_search_parser = vlv_subcommands.add_parser('list', help='List VLV search and index entries')
     list_vlv_search_parser.set_defaults(func=backend_list_vlv)
-    list_vlv_search_parser.add_argument('--just-names', action='store_true', help='List just the names of the VLV search entries')
+    list_vlv_search_parser.add_argument('--just-names', action='store_true', help='Displays only the names of VLV search entries')
     list_vlv_search_parser.add_argument('be_name', help='The backend name of the VLV index')
 
     # Get VLV search entry and indexes
-    get_vlv_search_parser = vlv_subcommands.add_parser('get', help='Get a VLV search & index')
+    get_vlv_search_parser = vlv_subcommands.add_parser('get', help='Display a VLV search and indexes')
     get_vlv_search_parser.set_defaults(func=backend_get_vlv)
-    get_vlv_search_parser.add_argument('--name', help='Get the VLV search entry and its index entries')
+    get_vlv_search_parser.add_argument('--name', help='Displays the VLV search entry and its index entries')
     get_vlv_search_parser.add_argument('be_name', help='The backend name of the VLV index')
 
     # Create VLV Search
-    add_vlv_search_parser = vlv_subcommands.add_parser('add-search', help='Add a VLV search entry.  The search entry is the parent entry '
-                                                                             'of the VLV index entries, and it specifies the search params that '
+    add_vlv_search_parser = vlv_subcommands.add_parser('add-search', help='Add a VLV search entry. The search entry is the parent entry '
+                                                                             'of the VLV index entries, and it specifies the search parameters that '
```

Comment:
Would you mind fixing the indentation of the help comment?  Thanks!

Reply:
done